src/sentry/eventstore/base.py

```diff
@@ -66,6 +66,18 @@ class Columns(Enum):
     CONTEXTS_VALUE = "contexts.value"
 
 
+class Filter(object):
+    def __init__(
+        self, start=None, end=None, conditions=None, project_id=None, group_id=None, event_id=None
```

Comment:
I believe project_id and group_id are sequences, so let's rename them to project_ids and group_ids.

Reply:
Renaming the keys will make moving from the current implementation to the new one a bit harder though.

Reply:
I was trying to avoid renaming too much, but the current implementation uses "issue" instead of "group_id" so maybe it's inevitable

Reply:
Renamed